core/src/main/java/io/questdb/cairo/map/Map.java

```diff
@@ -58,13 +58,15 @@ default long getUsedHeapSize() {
         return -1;
     }
 
+    boolean isOpen();
+
     void merge(Map srcMap, MapValueMergeFunction mergeFunc);
 
     /**
      * Reopens previously closed map with given key capacity and initial heap size.
      * Heap size value is ignored if the map does not use heap to store keys and values.
      */
-    void reopen(int keyCapacity, int heapSize);
+    void reopen(int keyCapacity, long heapSize);
```

Comment:
implementations are inconsistent. 2Map ignores the arguments. Also, naming is inconsistent, sometimes we mix `heapSize` and `memSize`

Reply:
All `UnorderedMap`s don't use heap, so as the javadoc says "Heap size value is ignored if the map does not use heap to store keys and values."

Reply:
Improve the javadoc in 20f8234b2e63bfeb67cd24e91355f6edbf1e3b89

> Also, naming is inconsistent, sometimes we mix `heapSize` and `memSize`

Could you give an example of such place?

Reply:
sure, OrderedMap has `heapSize`, all unordered `memSize`